File: Laboratorio hackeramk/02 JavaScript/02_constructor.js
class Jugador {
    constructor(nombre, nivel) {
        this.nombre = nombre;
        this.nivel = nivel;
    }

    mostrar() {
        console.log(this.nombre + " nivel " + this.nivel);
    }
}

const jugador = new Jugador("Pedro", 5);
jugador.mostrar();

File: Laboratorio hackeramk/04 C++/02_constructor.cpp
#include <iostream>
using namespace std;

class Jugador {
    string nombre;
    int nivel;

public:
    Jugador(string n, int l) {
        nombre = n;
        nivel = l;
    }

    void mostrar() {
        cout << nombre << " nivel " << nivel << endl;
    }
};

int main() {
    Jugador jugador("Pedro", 5);
    jugador.mostrar();
    return 0;
}
